DFC.App.JobProfile.CurrentOpportunities/Program.cs

```diff
@@ -1,17 +1,26 @@
 ﻿using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.ApplicationInsights;
 
 namespace DFC.App.JobProfile.CurrentOpportunities
 {
-    public class Program
+    public static class Program
     {
         public static void Main(string[] args)
         {
             CreateWebHostBuilder(args).Build().Run();
         }
 
-        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>();
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args)
+        {
+            return WebHost.CreateDefaultBuilder(args)
+                          .UseApplicationInsights()
+                          .ConfigureLogging((webHostBuilderContext, loggingBuilder) =>
+                          {
+                              loggingBuilder.AddFilter<ApplicationInsightsLoggerProvider>(string.Empty, LogLevel.Trace);
```

Comment:
How do we make it configurable? As we would not want to leave it in trace mode for ever and we may need to configure "Info" in dev, "Warning" in SIT  and "Error" in PP and PROD

Reply:
This will need to be done as a separate ticket across all apps as this the same in all of them at the moment.